resources/scripts/coverage.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```

Comment:
What is calling this script? I haven't seen a reference yet

Reply:
Ups! I pushed the wrong file to the wrong branch

Reply:
I just reverted the last three commits that I pushed wrongly